.rubocop.yml

```diff
@@ -13,17 +13,24 @@ Documentation:
 
 Metrics/AbcSize:
   Max: 16
+
+Metrics/BlockLength:
+  Exclude:
+    - 'spec/**/*.rb'
+
 Metrics/ClassLength:
   Max: 256
+
 Metrics/LineLength:
   Max: 120
   Exclude:
     - spec/**/*
+
 Metrics/MethodLength:
   Max: 15
 
-Rails:
```

Comment:
not sure why I needed to remove this but rubocop wouldn't run locally with this argument.

Probably a version conflict on my side but we need to do a major upgrade and then write out our rules file. It's a big task.